var/spack/repos/builtin/packages/r-haven/package.py

```diff
@@ -25,10 +25,13 @@ class RHaven(RPackage):
     depends_on('r@3.2:', when='@2.1.1:', type=('build', 'run'))
     depends_on('r-forcats@0.2.0:', type=('build', 'run'))
     depends_on('r-hms', type=('build', 'run'))
-    depends_on('r-rcpp@0.11.4:', type=('build', 'run'))
     depends_on('r-readr@0.1.0:', type=('build', 'run'))
     depends_on('r-rlang@0.4.0:', when='@2.3.1:', type=('build', 'run'))
     depends_on('r-tibble', type=('build', 'run'))
     depends_on('r-tidyselect', when='@2.3.1:', type=('build', 'run'))
     depends_on('r-vctrs@0.3.0:', when='@2.3.1:', type=('build', 'run'))
+    depends_on('r-cpp11', when='@2.4:', type=('build', 'run'))
     depends_on('gmake', type='build')
+    depends_on('zlib', when='@2.4:', type='build')
```

Comment:
This would be needed for linking as well.
```suggestion
    depends_on('zlib', when='@2.4:')
```